### PR TITLE
Data distribution warning

### LINKED ISSUE #2375: Using Data Distribution using NServiceBus
In a discussion earlier today with @DavidBoike and @udidahan and others in the unconference session, @udidahan mentioned that Data Distribution is a different style of architecture. 

Step 1: Add a disclaimer in the sample: 
https://docs.particular.net/samples/routing/data-distribution/
https://docs.particular.net/nservicebus/azure/data-distribution

~~Step 2: Come up with a general guidance for the comparison on both.~~ *Covered by #2410*

cc / @SzymonPobiega

### PR DESCRIPTION
**Pull request by DavidBoike**
Fixes #2375 

Just warnings

@indualagarsamy

### CODE REVIEW

File: nservicebus/azure/data-distribution.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ redirects:
 - nservicebus/scalability-and-ha/individualizing-queues-when-scaling-out
 ---
 
+DANGER: Asynchronous messaging is not an optimal solution for data distribution scenarios. It is usually better to use a dedicated data distribution technology for that purpose, such as a distributed cache or distributed configuration service.
```
Comment (indualagarsamy) [inline nservicebus/azure/data-distribution.md]: @yvesgoeleven and @SeanFeldman - Can you review this disclaimer to make sure that it is still accurate when it comes to Azure. Or can you see if we can make this disclaimer better by adding some more, especially when it comes to caching as a service and azure.

**Comment by SeanFeldman** [comment]
Also, is that a `danger` or a `warning`.

**Comment by DavidBoike** [comment]
@SeanFeldman Danger -> Red

File: nservicebus/azure/data-distribution.md (hunk 1 above)
Comment (SeanFeldman) [inline nservicebus/azure/data-distribution.md]: @yvesgoeleven is there anything you'd like to add?

**Pull request merged**